runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,20 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=%2$.*1$s has been compiled by a more recent version of the Java Runtime (class file version %3$u.%4$u), this version of the Java Runtime only recognizes class file versions up to %5$u.0; offset=%6$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.explanation=The major version in the classfile is unsupported on the VM rejecting the classfile.
+J9NLS_CFR_ERR_MAJOR_VERSION2.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MAJOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+
+# END NON-TRANSLATABLE
+
+
+J9NLS_CFR_ERR_MINOR_VERSION2=bad minor verion of class %2$.*1$s. Got minor verion .%4$u, but .0 is the only valid minor version for the major version %3$u; offset=%5$u
```

Comment:
Question for discussion: do we need to mention about allowed preview version in error message? 
should include `max_allowed_version.65535` as part of valid

As we are refactoring the major / minor version message, we should also consider adding new preview message as well to tell user that they are using wrong jdk version / or didn't turn on `--enable-preview` flag.
FYI @pshipton

Reply:
ie.  something like
```c++
if (majorVersion == max_allowed && minorVersion == 65535 && no_preview_flag)
    // class major.minor is preview, please run with --enable-preview flag
elif (majorVersion < max_allowed && majorVersion > 55 && minorVersion == 65535)
    // class major.minor unsupported preview major version, runtime only allow max_allowed.65535
```

Reply:
There is already a message, but it could be improved like you suggest.
`J9NLS_CFR_ERR_PREVIEW_VERSION=Class file is a preview version but has the wrong major version or preview is not enabled.`

Reply:
so should I do anything on the preview version?

Reply:
yes, you can improve it like Jack suggested.